### DISCUSSION TITLE
Switching to another workspace should open a new tab by default

### DISCUSSION THREAD

**Discussion by SIVAH-AKASH** [zen-browser/desktop#1853] [Ideas]
I have two workspaces currently and both have pinned tabs. After I reopen the browser, the current workspace creates a new tab by default, however switching to the other workspace opens the top pinned tab by default. I would prefer an option to open a new tab by default instead on the second workspace.

**Comment by Axenide** [comment]
Weird, this is the default behavior on my end.

**Reply by SIVAH-AKASH** [reply]
By "reopen", I meant fully exiting and then opening the browser (vs minimizing it). Is that what you did as well?

**Reply by Axenide** [reply]
> By "reopen", I meant fully exiting and then opening the browser (vs minimizing it). Is that what you did as well?

Ah, I get you now. Sorry, I misunderstood. I thought you weren't getting a new tab when switching to an empty workspace.
I second your opinion then.

**Comment by SIVAH-AKASH** [comment]
This seems to have been fixed in either 1.0.1-a.11 or 1.0.1-a.12.
